gpMgmt/test/behave/mgmt_utils/gpinitsystem.feature

```diff
@@ -17,6 +17,28 @@ Feature: gpinitsystem tests
         And gpconfig should print "Master  value: off" to stdout
         And gpconfig should print "Segment value: off" to stdout
 
+    Scenario: gpinitsystem creates a cluster when the user set LC_ALL env variable
+        Given create demo cluster config
+        And the user runs command "export LC_ALL=en_US.UTF-8"
```

Comment:
Instead of specifying `export` command use more appropriate [action](https://github.com/arenadata/gpdb/blob/ba91667ba5698367d057b26c392447f6db72fbd2/gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py#L1377-L1380).